frontend/src/main/java/cz/cuni/mff/xrg/odcs/frontend/gui/tables/LogTable.java

```diff
@@ -172,9 +184,55 @@ public String generateDescription(Component source, Object itemId, Object proper
         table.setFilterBarVisible(true);
         table.setPageLength(pageLenght);
 
+        Button download = new Button(Messages.getString("DebuggingView.download"));
+        FileDownloader fileDownloader = new OnDemandFileDownloader(new OnDemandStreamResource() {
+            @Override
+            public String getFilename() {
+                return "log.csv";
+            }
+
+            @Override
+            public InputStream getStream() {
+
+                File fileToDownload;
+                try {
+                    fileToDownload = File.createTempFile("tempfile", "temp");
```

Comment:
http://docs.oracle.com/javase/7/docs/api/java/io/File.html#createTempFile(java.lang.String,%20java.lang.String)
*Creates an empty file in the default temporary-file directory, using the given prefix and suffix to generate its name.*